openapi.yml

```diff
@@ -210,14 +224,299 @@ paths:
+      responses:
+        '200':
+          description: A list of Load Balancers in a specific project
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/LoadBalancerDetailed'
+        default:
+          $ref: '#/components/responses/Error'
+      tags:
+        - Load Balancer
+    post:
+      operationId: createLoadBalancer
+      summary: Create a new Load Balancer in a project
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              type: object
+              properties:
+                algorithm:
+                  description: Algorithm of the Load Balancer
+                  type: string
+                dst_port:
+                  description: Destination port for the Load Balancer
+                  type: integer
+                health_check_endpoint:
+                  description: Health check endpoint URL
+                  type: string
+                health_check_protocol:
+                  description: Health check endpoint protocol
+                  type: string
+                private_subnet_id:
+                  description: ID of Private Subnet
+                  type: string
+                src_port:
+                  description: Source port for the Load Balancer
+                  type: integer
+              additionalProperties: false
+              required:
+                - algorithm
+                - dst_port
+                - health_check_protocol
+                - private_subnet_id
+                - src_port
+      responses:
+        '200':
+          description: Load Balancer created successfully
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/LoadBalancerDetailed'
+        default:
+          $ref: '#/components/responses/Error'
+      tags:
+        - Virtual Machine
+  '/project/{project_id}/location/{location}/firewall':
+    parameters:
+      - $ref: '#/components/parameters/location'
+      - $ref: '#/components/parameters/project_id'
+    get:
+      operationId: getLocationFirewall
+      summary: Return the list of firewalls in the project and location
+      responses:
+        '200':
+          description: Firewall list retrieved successfully
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  count:
+                    type: integer
+                  items:
+                    type: array
+                    items:
+                      $ref: '#/components/schemas/Firewall'
+                additionalProperties: false
+                required:
+                  - count
+                  - items
+        default:
+          $ref: '#/components/responses/Error'
+      tags:
+        - Firewall
+  '/project/{project_id}/location/{location}/firewall/id/{firewall_id}':
```

Comment:
Shouldn't this be `firewall/{firewall_name_or_id}` instead of `firewall/id/{firewall_id}`, since a5be9194193bedd821d93d8f682ac64826a3b1f9 has been merged? Ditto for some other similar paths.  Or is that change expected to be merged later?

Reply:
this predates that, but we'll probably snap the openapi.yml again. I suppose given the documentation repo is not  dependent on this copy, it can wait.